Task3/Task3-3/main.cpp

```diff
@@ -0,0 +1,54 @@
+// Task3-3.
+
+#include <iostream>
+#include <cmath>
+/**
+*\ brief  The volume of the Parallelepiped
+*\ param width,length,height
+*\ return volume
+*/
+double funct(double x);
+/**
+*\ brief  The area of the Parallelepiped
+*\ param width,length,height
+*\ return  area
+*/
+int fact(int k);
+double ryd(double x);
+int main()
+{
+    double x=0.1;
+    while (x < 1)
+    {
+    std::cout << "x="<<x<<" y="<<funct(x)<<" ryd="<<ryd(x)<<"\n";
+    x += 0.1;
+    }
+}
+double funct(double x)
+{
+    return exp(2*x);
+}
+double ryd(double x)
```

Comment:
Если параметры функции не изменяются, то они явно должны быть обозначены `const`
[By Alex Allain](https://www.cprogramming.com/tutorial/const_correctness.html)
>The const keyword allows you to specify whether or not a variable is modifiable. You can use const to prevent modifications to variables and const pointers and const references prevent changing the data pointed to (or referenced).
>Const дает вам возможность более четко документировать вашу программу и фактически обеспечивать соблюдение этой документации. Ключевое слово const обеспечивает вашим пользователям гарантии, позволяющие оптимизировать производительность без угрозы повреждения их данных, обеспечивая соблюдение вашей документации. Например, константные ссылки позволяют указать, что упомянутые данные не будут изменены; это означает, что вы можете использовать константные ссылки как простой и немедленный способ повышения производительности для любой функции, которая в настоящее время принимает объекты по значению, не беспокоясь о том, что ваша функция может изменить данные. Даже если это произойдет, компилятор предотвратит компиляцию кода и предупредит вас о проблеме. С другой стороны, если бы вы не использовали константные ссылки, у вас не было бы простого способа гарантировать, что ваши данные не были изменены.